### PR TITLE
Tutorial: How to Install mitmproxy-ca-cert.pem on Wii U

### PR DESCRIPTION
**Pull request by BluelyTASer**
#### Description

This Tutorial Explains on How to Install mitmproxy-ca-cert.pem on Wii U

#### Tutorial

First you download mitmproxy-ca-cert.pem on your computer
Then you need to homebrew your Wii U: https://wiiu.hacks.guide/#/ 
Then Install FTPiiU and run it on Homebrew Channel
Download /storage_mlc/usr/nsec/sys/wiiu_device_certchain.pem on your Wii U from your FTP Client
Open both mitmproxy-ca-cert.pem and wiiu_device_certchain.pem as notepad
Copy all text of mitmproxy-ca-cert.pem then append it on wiiu_device_certchain.pem and save it
Upload Your Modified wiiu_device_certchain.pem to a directory (/storage_mlc/usr/nsec/sys/) from your FTP Client
Restart Your Wii U and You're done!
Now it's capturing https traffic from your Wii U! 😄

#### Note
The Error Code 102-2160 that points to HTTP_SSL_CACERT will always be appeared while connected to Mitmproxy from your Wii U Console!
 
#### Photos of Proof
Picture

### CODE REVIEW

**Comment by BluelyTASer** [comment]
Video Testing:
[Video](https://www.youtube.com/watch?v=q3Y2HywSKz8)

File: README.md [hunk 1]
```diff
@@ -18,6 +17,8 @@ interface for HTTP/1, HTTP/2, and WebSockets.
 The installation instructions are [here](https://docs.mitmproxy.org/stable/overview-installation).
 If you want to install from source, see [CONTRIBUTING.md](./CONTRIBUTING.md).
 
+``Wii U instalation docs are avaliable, Homebrewed Wii U Required``
```
Comment (Kriechi) [inline README.md:20]: Where are they available?

**Review by Kriechi — CHANGES_REQUESTED** [review]
This looks incomplete...

File: README.md (hunk 1 above)
Comment (BluelyTASer) [reply README.md]: Fixed it from my next [commit](https://github.com/mitmproxy/mitmproxy/pull/4445/commits/e00d2395a60926d909419a5fe36386b94a9095a0), please let me know if it's still incomplete, if you think that's still incomplete, then I need to hire someone to make video tutorial on this, I don't want to brick my wii u just doing it myself over again.

**Comment by Bluely-NO-STALKING-MYSELF-ALLOWED** [comment]
more than a year since it's closed, i'm sorry but i should explain more in details next time

**Pull request closed without merging**